### ISSUE TITLE
updates?

### ISSUE THREAD

**Issue by shayanb** [doefler/Instagram-growth#1]
Does Instagram growth works with the recent changes in Instagram API ? in other words, is this still functional?

**Comment by shayanb** [comment]
e.g. https://www.instagram.com/developer/sandbox/

**Comment by doefler** [comment]
Hey @shayanb thank you for making me aware of the recent changes :+1: 
Unfortunately it seems like it might affect this tool, I will have to look more into it to see how much of a problem it is and how to fix it.
I am sorry I cannot provide you with a solution now...
Have a nice weekend :smile:

**Comment by shayanb** [comment]
Yeah I've been away from instagram api for almost 6 months now and now none of my code works anymore, and apparently if you want to get like/comment/... permission from a user you have to go through a review process of your "app" to be able to do so.
you too , have a good weekend :)

**Comment by PietroGranati** [comment]
Yep , I can like photos but not comment, it return:
Comment image: {"meta":{"error_type":"OAuthPermissionsException","code":400,"error_message":"This client has not been approved to access this resource."}}

**Comment by shayanb** [comment]
Instagram requires your app to be verified to be able to ask the user for most of the permissions.

> Instagram Platform and documentation update. Apps created on or after Nov 17, 2015 will start in Sandbox Mode and function on newly updated API rate-limits and behaviors. Prior to going Live, and being able to be used by people other than the developers of the app, these apps will have to go through a new review process. Please read the API documentation or the Change Log for more details.

> Any app created before Nov 17, 2015 will continue to function until June 2016. After June 2016, the app will automatically be moved to Sandbox Mode if it wasn't approved through the review process. The previous version of our documentation is still available here.
> https://www.instagram.com/developer/authorization/

does anyone have any ideas how to get this working though? maybe an app that got access before these dates?

**Comment by manumanu2000** [comment]
LevPasha seems to have solved this problem in his package (https://github.com/LevPasha/instabot.py) i think this is a good step for a new api aggiration

**Comment by shayanb** [comment]
nice, thank, will test it out over the weekend.
